executorWdl/src/main/scala/dx/executor/wdl/WdlWorkflowExecutor.scala

```diff
@@ -46,7 +46,7 @@ object WdlWorkflowExecutor {
     val tasks = doc.elements.collect {
       case task: TAT.Task => task.name -> task
     }.toMap
-    WdlWorkflowExecutor(workflow, versionSupport, tasks, typeAliases.toMap, jobMeta)
+    WdlWorkflowExecutor(doc.source, workflow, versionSupport, tasks, typeAliases.toMap, jobMeta)
```

Comment:
Would it be possible to add a short docstring, explaining esp what doc & docSource are?

Reply:
done